src/app/[locale]/projects/detail/[id]/components/AttachmentUsages.tsx

```diff
@@ -145,184 +208,87 @@ export default function AttachmentUsages({ projectId }: { projectId: string }) {
                     if(proj.length === 0) {
                         continue
                     }
-                    const releases = new Set<string>();
+                    const releases = new Set<string>()
                     proj[0].linkedReleases?.map((r: any) => {
-                        releases.add(
-                            r.release.substring(
-                                r.release.lastIndexOf('/') + 1
-                            )
-                        )
+                        releases.add(r.release.split('/').at(-1))
                     })
                     const node: NodeData = {
                         rowData: [
-                            <Link key={`${proj[0]["_links"]["self"]["href"].substring(
-                                    proj[0]["_links"]["self"]["href"].lastIndexOf('/') + 1
-                                )}_name_project`} 
-                                href={`/projects/detail/${proj[0]["_links"]["self"]["href"].substring(
-                                proj[0]["_links"]["self"]["href"].lastIndexOf('/') + 1
-                            )}`}>
+                            <Link key={`${proj[0]["_links"]["self"]["href"].split('/').at(-1)}_name_project`} 
+                                href={`/projects/detail/${proj[0]["_links"]["self"]["href"].split('/').at(-1)}`}>
                                 {`${proj[0].name ?? ''} ${proj[0].version ?? ''}`}
                             </Link>,
-                            <div key={`${proj[0]["_links"]["self"]["href"].substring(
-                                proj[0]["_links"]["self"]["href"].lastIndexOf('/') + 1
-                            )}_relationship_project`}>
+                            <div key={`${proj[0]["_links"]["self"]["href"].split('/').at(-1)}_relationship_project`}>
```

Comment:
`${proj[0]["_links"]["self"]["href"].split('/').at(-1)}`
This code snippet is duplicated at many places, could you replace them by ${projectId}?